packit_service/worker/jobs.py

```diff
@@ -483,6 +483,18 @@ def get_jobs_matching_event(self) -> List[JobConfig]:
             ):
                 jobs_matching_trigger.append(job)
 
+        if not jobs_matching_trigger and isinstance(
+            self.event, PullRequestCommentPagureEvent
+        ):
+            if (
+                job.type == JobType.koji_build
+                and job.trigger == JobConfigTriggerType.commit
+                and self.event.job_config_trigger_type
+                == JobConfigTriggerType.pull_request
+            ):
```

Comment:
I think this wouldn't work since we don't run this condition for all jobs in a cycle :thinking:..but looking into Steve code, couldn't we use [`get_handlers_for_comment`](https://github.com/packit/packit-service/blob/da4a797985708b814e744a7c19610f00384d6729/packit_service/worker/jobs.py#L64) (which uses the mapping via `@run_for_comment` decorator you correctly used) instead of hardcoding this entirely? (maybe some hardcoding will still be needed, but it should make things simpler)

Reply:
ok, I understand :+1: